engine/schema/src/main/resources/META-INF/db/schema-41910to42000.sql

```diff
@@ -425,3 +425,9 @@ INSERT IGNORE INTO `cloud`.`guest_os_hypervisor` (uuid, hypervisor_type, hypervi
 
 CALL `cloud`.`IDEMPOTENT_ADD_COLUMN`('cloud.vm_instance', 'delete_protection', 'boolean DEFAULT FALSE COMMENT "delete protection for vm" ');
 CALL `cloud`.`IDEMPOTENT_ADD_COLUMN`('cloud.volumes', 'delete_protection', 'boolean DEFAULT FALSE COMMENT "delete protection for volumes" ');
+
```

Comment:
targeted for 4.20.1? add these changes in _engine/schema/src/main/resources/META-INF/db/schema-42000to42010.sql_

Reply:
Will create the new schema file once 4.20.0 is cut. Keeping this PR in draft until then